content/hi/docs/reference/glossary/uid.md

```diff
@@ -0,0 +1,15 @@
+---
+title: यूआईडी (UID)
+id: uid
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/names
+short_description: >
+  विशिष्ट वस्तुओं की पहचान करने के लिए एक कुबेरनेट्स सिस्टम-जनित स्ट्रिंग।
+aka: 
+tags:
+- fundamental
+---
+ विशिष्ट वस्तुओं की पहचान करने के लिए एक कुबेरनेट्स सिस्टम-जनित स्ट्रिंग।
+<!--more--> 
+
+एक कुबेरनेट्स क्लस्टर के पूरे जीवनकाल में बनाई गई प्रत्येक वस्तु का एक अलग UID है। इसका उद्देश्य समान संस्थाओं की ऐतिहासिक घटनाओं के बीच अंतर करना है।
```

Comment:
```suggestion
एक कुबेरनेट्स क्लस्टर के पूरे जीवनकाल में बनाई गई प्रत्येक ऑब्जेक्ट का एक अलग UID होता है। इसका उद्देश्य समान इकाइयों की ऐतिहासिक घटनाओं के बीच अंतर करना है।
```

Reply:
@anubha-v-ardhan  All the suggested changes have been applied now.